config/config.msft.yaml

```diff
@@ -112,8 +112,11 @@ defaults:
   # Monitoring
   monitoring:
     grafanaName: "arohcp-{{ .ctx.environment }}"
-    # Format: Multiline string using '>-' YAML block scalar
+    # Format:
```

Comment:
@geoberle @tony-schndr given that I did not edit the config myself in this PR, I'm wondering how this diff snuck in and didn't trip the CI?